save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityCollectionView.kt

```diff
@@ -80,7 +80,7 @@ val vulnerabilityCollectionView: FC<VulnerabilityCollectionViewProps> = FC { pro
                             }
                         }
                     }
-                    column(id = "short_description", header = "Description".t(), { progress }) { cellContext ->
+                    column(id = "short_description", header = "Description".t(), { description }) { cellContext ->
```

Comment:
maybe change id then?

Reply:
oh sure, will do it

Reply:
Changed `description` to `shortDescription`